processing/deps/rasters_to_netcdf.py

```diff
@@ -0,0 +1,182 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import List, Union
+
+import numpy as np
+import netCDF4
+from osgeo import gdal, osr
+from pyproj import CRS
+
+gdal.UseExceptions()
+osr.UseExceptions()
+
+
+def get_datasets(filepaths: List[Union[str, Path]]) -> List[gdal.Dataset]:
+    result = list()
+    for filepath in filepaths:
+        dataset = gdal.Open(str(filepath))
+        if not dataset:
+            raise Exception(f"Unable to open {filepath}")
+        result.append(dataset)
+    return result
```

Comment:
Dit plaatst alle rasters in memory, misschien dat gdal nog iets speciaals doet waardoor de impact gereduceerd is, maar als iemand een flinke set aan rasters wil transformen naar netcdf gaat dit denk ik wel problemen opleveren.

Reply:
Net nog ff bij Arjan gecheckt, maar een `gdal.Dataset` neemt amper geheugen in beslag en is puur een verwijzing naar het raster; dus op deze plek in de code is er nog geen memory issue